### PR TITLE
Fix crash in splash when offline

### PR DESCRIPTION
**Pull request by ogapants**
## Issue
none

## Overview (Required)
Since the app has no error handling in `loadSessionsForCache()`, it crashes when fails to load sessions in splash. 💥 
I added only logs using Timber.
I think that we need to implement session reloading function in the future.

## Links
none

## Screenshot
none

### CODE REVIEW

**Comment by konifar** [comment]
:one: :one:

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java [hunk 1]
```diff
@@ -73,7 +71,8 @@ private void loadSessionsForCache() {
                     if (isFinishing()) return;
                     startActivity(MainActivity.createIntent(SplashActivity.this));
                 })
-                .subscribe();
+                .subscribe(objectObservable -> Timber.tag(TAG).d("Succeeded in loading sessions."),
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java]: Can you change valuable to `observable` from `objectObservable`?
In fact, lambda variable should be more short name like `o`, `t`.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java]: `Succeeded to load sessions.` is better?

Comment (ogapants) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java]: I'll fix it soon! Thanks 🙇

Comment (ogapants) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java]: I think that "succeed to" means "引き継ぐ"... 🤔

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java]: Oh I didn't know that 🙇 Thanks

**Comment by konifar** [comment]
LGTM! 💯

**Pull request merged**